postgres/assets/configuration/spec.yaml

```
@@ -145,6 +145,13 @@ files:
       value:
         type: integer
         example: 60000
+    - name: max_connections
+      description: |
+        Sets the maximum number of connections the agent can make to the Postgres server concurrently.
```

Small nit: this description will potentially look dangerous to users who do not understand the intent. I will look at that and think "What?? Datadog is going to open 30 constant connections to my database?" When in fact that is not the case for 99.99% of users. This really is an advanced feature for advanced users or customers support is working with, but is merely a safety valve for everyone else.

I suggest two small changes:

1.) Make the option hidden to avoid users discovering it and misinterpreting the value
2.) Adjust the description with 
```suggestion
        Sets the upper bound limit on the maximum number of connections the agent can concurrently have on the database server. The Agent needs to open additional connections to collect explain plans across databases. Idle connections will be pruned over time to balance performance impact of opening connections and the cost of having many open connections. Raising the default value should only be needed to be raised on workloads with >50 databases. Lowering the default value can affect the number of explain plans collected.
```

---

yeah, that makes a lot of sense! i will update the description